<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWelcome"/>
   <addaction name="actionMonitoring"/>
   <addaction name="actionEvents"/>
   <addaction name="actionNotify"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNotify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Notify</string>
   </property>
  </action>
  <action name="actionWelcome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </action>
  <action name="actionMonitoring">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Monitoring</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Events</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
